lib/wibox/widget/slider.lua

```diff
@@ -565,7 +572,9 @@ end
 -- @tparam[opt] number args.handle_width The slider handle width.
 -- @tparam[opt] color args.handle_border_color The handle border_color.
 -- @tparam[opt] number args.handle_border_width The handle border width.
--- @tparam[opt] string args.handle_cursor The handle grab cursor.
+-- @tparam[opt] string args.handle_cursor
+-- The cursor icon while grabbing the handle.
+-- The available cursor names are listed under handle_cursor, in the "Object properties" section.
```

Comment:
You should add a few (2 or 3 I don't remember) extra spaces leading the additional lines

Reply:
Fixed. Thanks for the feedback!